### DISCUSSION TITLE
Freeplane stopped working

### DISCUSSION THREAD

**Discussion by liusiguang** [freeplane/freeplane#1735] [Help]
Freeplane loads (shows up in Task Manager) but the interface does not appear. Have downloaded the latest update - same results. 
The last time I used it, all worked as expected. Now I get nothing displayed. The most likely reason is that something else is blocking the display part of FP. 
Has anyone had a similar experience and, if so, was it resolved?

**Comment by i-plasm** [comment]
Hello @liusiguang , I recommend posting an issue: https://github.com/freeplane/freeplane/issues

**Reply by liusiguang** [reply]
Thank you.

**Comment by dpolivaev** [comment]
Rename or delete the directory containing yourFreeplane user settings and check if it starts if all user options are reset.

**Reply by liusiguang** [reply]
I am not familiar with the freeplane layout. Please provide the path to user settings for a Windows 11 installation.
Thank  you.

**Reply by dpolivaev** [reply]
C:\Users\Watashi\AppData\Roaming\Freeplane\1.11.x

**Comment by dpolivaev** [comment]
You can also run freeplaneConsole.exe from the installation directory and post here the output from the console.

**Reply by liusiguang** [reply]
Below are the results of running freeplaneConsole.exe
-------------------
Knopflerfish OSGi framework launcher, version <unknown>
Copyright 2003-2020 Knopflerfish. All Rights Reserved.
See http://www.knopflerfish.org for more information.

Created Framework: org.knopflerfish.framework, version=8.0.11.
Background mode.
Mar 12, 2024 3:34:59 PM org.freeplane.core.util.LogUtils info
INFO: Loaded properties from bundle://1:1/freeplane.properties
Mar 12, 2024 3:34:59 PM org.freeplane.core.util.LogUtils info
INFO: Loaded properties from bundle://1:2/org/freeplane/main/application/
Mar 12, 2024 3:34:59 PM org.freeplane.core.util.LogUtils info
INFO: Listening for application instances on socket 53626
Installed: reference:file:C:\Program Files\Freeplane\plugins\org.freeplane.plugin.bugreport (id#2)
Installed: reference:file:C:\Program Files\Freeplane\plugins\org.freeplane.plugin.formula (id#3)
Installed: reference:file:C:\Program Files\Freeplane\plugins\org.freeplane.plugin.jsyntaxpane (id#4)
Installed: reference:file:C:\Program Files\Freeplane\plugins\org.freeplane.plugin.latex (id#5)
Installed: reference:file:C:\Program Files\Freeplane\plugins\org.freeplane.plugin.markdown (id#6)
Installed: reference:file:C:\Program Files\Freeplane\plugins\org.freeplane.plugin.openmaps (id#7)
Installed: reference:file:C:\Program Files\Freeplane\plugins\org.freeplane.plugin.script (id#8)
Installed: reference:file:C:\Program Files\Freeplane\plugins\org.freeplane.plugin.svg (id#9)
Started: reference:file:C:\Program Files\Freeplane\plugins\org.freeplane.plugin.bugreport (id#2)
Started: reference:file:C:\Program Files\Freeplane\plugins\org.freeplane.plugin.formula (id#3)
Started: reference:file:C:\Program Files\Freeplane\plugins\org.freeplane.plugin.jsyntaxpane (id#4)
Started: reference:file:C:\Program Files\Freeplane\plugins\org.freeplane.plugin.latex (id#5)
Started: reference:file:C:\Program Files\Freeplane\plugins\org.freeplane.plugin.markdown (id#6)
Started: reference:file:C:\Program Files\Freeplane\plugins\org.freeplane.plugin.openmaps (id#7)
Started: reference:file:C:\Program Files\Freeplane\plugins\org.freeplane.plugin.script (id#8)
Started: reference:file:C:\Program Files\Freeplane\plugins\org.freeplane.plugin.svg (id#9)
Mar 12, 2024 3:34:59 PM org.freeplane.core.util.LogUtils info
INFO: language resources for en found
Mar 12, 2024 3:34:59 PM org.freeplane.core.util.LogUtils info
INFO: freeplane_version = 1.11.4; freeplane_xml_version = freeplane 1.11.1
git revision = 62e12866e72e5e50519962fccfc06244c3cf4b59 dpolivaev 2023-05-28T13:36:02Z
java_version = 15.0.10; os_name = Windows 11; os_version = 10.0
Mar 12, 2024 3:34:59 PM org.freeplane.core.util.LogUtils info
INFO: added parsing support for standard date format M/d/yy
Mar 12, 2024 3:34:59 PM org.freeplane.core.util.LogUtils info
INFO: added parsing support for standard date time format M/d/yy, h:mm a
Framework launched
Mar 12, 2024 3:35:01 PM org.freeplane.core.util.LogUtils info
INFO: Loaded properties from bundle://3:1/org/freeplane/plugin/formula/defaults.properties
Mar 12, 2024 3:35:01 PM org.freeplane.core.util.LogUtils info
INFO: extending script's classpath by [C:\Users\Watashi\AppData\Roaming\Freeplane\1.11.x\compiledscripts, C:\Users\Watashi\AppData\Roaming\Freeplane\1.11.x\lib]
Mar 12, 2024 3:35:01 PM org.freeplane.core.util.LogUtils info
INFO: Loaded properties from bundle://8:1/org/freeplane/plugin/script/defaults.properties
Mar 12, 2024 3:35:01 PM org.freeplane.core.util.LogUtils info
INFO: looking for scripts with the following endings: [groovy]
Mar 12, 2024 3:35:01 PM org.freeplane.core.util.LogUtils info
INFO: looking for scripts with the following endings: [groovy]
Mar 12, 2024 3:35:02 PM org.freeplane.core.util.LogUtils info
INFO: requesting mode: MindMap
Mar 12, 2024 3:35:04 PM org.freeplane.core.util.LogUtils info
INFO: menu items to execute: []
----------------------------------
FreeplaneConsole stopped at this point.

**Comment by liusiguang** [comment]
Thank you, sir.
All seems to be working as expected. I will keep this fix in my "Things to Remember" kit.
LSG






On Wednesday, March 13, 2024 at 06:17:24 AM EDT, Dimitry Polivaev ***@***.***> wrote: 







C:\Users\Watashi\AppData\Roaming\Freeplane\1.11.x
